ironfish/src/rpc/adapters/tcpAdapter.test.ts

```diff
@@ -53,7 +56,7 @@ describe('TcpAdapter', () => {
       request.end(request.data)
     })
 
-    client = new RpcTcpClient('localhost', tcp.addressPort)
+    client = new RpcTcpClient('localhost', tcp.addressPort, 'test token')
```

Comment:
I think we shouldn't pass this in these tests. We should test that we don't need to pass this in the old tests to show that they still pass.